gazebo/rendering/CMakeLists.txt

```diff
@@ -134,6 +137,9 @@ set (headers
   WrenchVisual.hh
   ogre_gazebo.h
 )
+if (${OGRE_VERSION} VERSION_LESS 1.10)
+  list(APPEND headers "CustomAutoParamDataSource.hh")
```

Comment:
does `gazebo/rendering/CustomSceneManager.hh` need to be installed?

Reply:
It works without it, but it doesn't hurt to add it for completeness. Done.